sdk/tables/azure-data-tables/azure/data/tables/_policies.py

```diff
@@ -353,19 +354,64 @@ def on_response(self, request, response):
                 )
 
 
-class StorageRetryPolicy(HTTPPolicy):
+class StorageRetryPolicy(RetryPolicy):
     """
-    The base class for Exponential and Linear retries containing shared code.
+    A base class for retry policies for the Table Client and Table Service Client
     """
+    def __init__(
+        self,
+        initial_backoff=15, # type: int
+        increment_base=3, # type: int
+        retry_total=10, # type: int
+        retry_to_secondary=False, # type: bool
+        random_jitter_range=3, # type: int
+        **kwargs # type: Any
+    ):
+        """
+        Build a BLANK retry object.
 
-    def __init__(self, **kwargs):
-        self.total_retries = kwargs.pop('retry_total', 10)
+        :param int initial_backoff:
+            The initial backoff interval, in seconds, for the first retry.
+        :param int increment_base:
+            The base, in seconds, to increment the initial_backoff by after the
+            first retry.
+        :param int max_attempts:
+            The maximum number of retry attempts.
+        :param int retry_total: total number of retries
+        :param bool retry_to_secondary:
+            Whether the request should be retried to secondary, if able. This should
+            only be enabled of RA-GRS accounts are used and potentially stale data
+            can be handled.
+        :param int random_jitter_range:
+            A number in seconds which indicates a range to jitter/randomize for the back-off interval.
+            For example, a random_jitter_range of 3 results in the back-off interval x to vary between x+3 and x-3.
+        """
+        self.initial_backoff = initial_backoff
+        self.increment_base = increment_base
+        self.random_jitter_range = random_jitter_range
+        self.total_retries = retry_total
         self.connect_retries = kwargs.pop('retry_connect', 3)
         self.read_retries = kwargs.pop('retry_read', 3)
         self.status_retries = kwargs.pop('retry_status', 3)
-        self.retry_to_secondary = kwargs.pop('retry_to_secondary', False)
+        self.retry_to_secondary = retry_to_secondary
```

Comment:
I think we should at least pass the kwargs to the parent `RetryPolicy`, Doesn't really look like there's much of an overlap though between the parameters fro `StorageRetryPolicy` and `RetryPoliccy`